package.json

```diff
@@ -87,6 +87,7 @@
     "typescript": "^3.8.3",
     "webpack": "^4.42.0",
     "webpack-cli": "^3.3.11",
+    "webpack-node-externals": "^1.7.2",
     "winston": "^3.2.1",
```

Comment:
@danbilokha Just curious is it ok to have in dev deps those pakages we will need in prod?

Reply:
dev deps will not be present on prod (at least I think so, it's the way I expect it to work)